features/conversation/hooks/use-react-on-message.ts

```diff
@@ -8,56 +9,61 @@ import { captureError, captureErrorWithToast } from "@/utils/capture-error";
 import { getTodayNs } from "@/utils/date";
 import { getRandomId } from "@/utils/general";
 import { Haptics } from "@/utils/haptics";
-import { ConversationWithCodecsType } from "@/utils/xmtpRN/client.types";
 import { contentTypesPrefixes } from "@/utils/xmtpRN/content-types/content-types";
 import { useMutation } from "@tanstack/react-query";
 import {
+  ConversationTopic,
   MessageDeliveryStatus,
   MessageId,
   ReactionContent,
 } from "@xmtp/react-native-sdk";
 import { useCallback } from "react";
 
-export function useReactOnMessage(props: {
-  conversation: ConversationWithCodecsType;
-}) {
-  const { conversation } = props;
+export function useReactOnMessage(props: { topic: ConversationTopic }) {
```

Comment:
in programming always better to interact by id rather than object - excellent stuff here!